src/components/notifications/AppSelector/index.tsx

```diff
@@ -173,6 +168,10 @@ const AppSelector: React.FC = () => {
                   </AnimatePresence>
                 )
               })}
+            {loading || !subscriptionsFinishedLoading? Array(3)
```

Comment:
Not urgent but we should create the array on the top of the component so it doesn't have to be recreated on every render: 

```jsx
const LoadingItems = Array(3).Fill(<LinkItemSkeleton />).map(x => x)
```

Reply:
Yes, let's handle this in a different PR. I really want to handle everything a bit faster